airbyte-cdk/python/airbyte_cdk/models/airbyte_protocol.py

```diff
@@ -244,6 +257,16 @@ class Config:
     )
 
 
+class AirbyteStateMessage(BaseModel):
+    class Config:
+        extra = Extra.allow
+
+    state_type: Optional[AirbyteStateType] = None
+    data: Optional[Dict[str, Any]] = Field(None, description="(Deprecated) the state data")
+    global_: Optional[AirbyteStateBlob] = Field(None, alias="global")
```

Comment:
@pedroslopez the `global_` is correct?

Reply:
Nice catch! Yeah... `global` is a reserved word in python, so the generator added the underscore, making it `global_`.